ext/sysvshm/tests/003.phpt

```diff
@@ -36,6 +28,5 @@ shm_remove($s);
 ?>
 --EXPECT--
 bool(true)
-shm_detach(): supplied resource is not a valid sysvshm resource
-shm_remove(): supplied resource is not a valid sysvshm resource
+bool(true)
```

Comment:
I'd expect both of these exceptions to be retained.

Reply:
They are retained now. I also applied the same checks for the other functions as well.